native/fastlane/read_build_config.rb

```diff
@@ -1,11 +1,6 @@
 require "json"
 
-def read_build_config(build_config_name, platform, project_root='.')
-  json = yarn(
-      command: "babel-node tools/build-config to-json #{build_config_name} #{platform}",
-      project_root: project_root,
-      flags: "--silent"
-  )
-
+def read_build_config(build_config_name, platform)
+  json = `yarn workspace --silent build-configs --silent manage to-json #{build_config_name} #{platform}`
```

Comment:
Crazy, how does it now it should be executed?

Reply:
backtick magic :D
thats how things work in ruby: magic

Reply:
Not sure if I like it :thinking:

Reply:
github does not show the highlighting correct, looks different in intellij